documentation/src/main/wiki/Dictionary.md

```diff
@@ -28,7 +28,7 @@ usually used in the context of a small piece of data, the result of some operati
 see *Amazon Web Services*.
 
 *Baseline*  
-The act of taking a snapshot of the url/page and saving it to a file for future comparison in a number of ways to find differences.
+the act of taking a snapshot of the url/page and saving it to a file for its future comparison in a number of ways to find differences.
```

Comment:
Still doesn't sound quite right. Correct me if I'm wrong but _baseline_, as a noun, is not the act of taking a snapshot. It's the snapshot itself. 

If, on the other hand, we're liberal enough to use the word _baseline_  as a verb, I believe _the act of taking a snapshot_ would be... _baselining_? I suggest we just stick with the noun :)

How about: _a snapshot of the page, stored in a file, against which future versions will be compared._